priv/graphql/schemas/user/private.gql

```diff
@@ -0,0 +1,13 @@
+"""
+Allow user to set own private
+"""
+type PrivateUserMutation @protected {
+    setPrivate(privateString: String!): String
```

Comment:
This looks like it could be any string, while it needs to be valid XML. Maybe `elementString`?